ducktape/src/main/scala/io/github/arainko/ducktape/internal/modules/MaterializedConfiguration.scala

```diff
@@ -90,6 +90,12 @@ private[ducktape] object MaterializedConfiguration {
         val sourceFieldName = Selectors.fieldName(Fields.source, sourceSelector)
         Product.Renamed(destFieldName, sourceFieldName) :: Nil
 
+      case '{ FieldConfig.default[source, dest, destFieldType]($destSelector)(using $ev1, $ev2) } =>
+        val destFieldName = Selectors.fieldName(Fields.dest, destSelector)
+        val field = Fields.dest.byName.get(destFieldName).getOrElse(Failure.abort(Failure.InvalidFieldSelector(destSelector, summon, Suggestion.fromFields(Fields.dest))))
```

Comment:
I think you can just do `Fields.dest.unsafeGet`, `Selectors.fieldName(Fields.dest, destSelector)` already does the check you do here` so I think it'll be safe